CHANGES/5184.misc

```diff
@@ -0,0 +1 @@
+Adding `code` on `ProgressBar`.
```

Comment:
Nitpick should probably be "Setting code..." rather than "Adding code...".

Also, I'd probably consider this a feature?